cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorCustomCertTest.java

```diff
@@ -153,197 +231,207 @@ public Pod getPod(StatefulSet sts) {
+                assertThat(isPodToRestart.apply(getPod(reconcileSts)), is(nullValue()));
+
+                async.flag();
+            })));
     }
 
-    @Test
-    public void testWithChangedTlsCertificate(VertxTestContext context) throws InterruptedException {
-        Kafka kafka = getKafkaCrd();
-        ResourceOperatorSupplier supplier = ResourceUtils.supplierWithMocks(false);
-        KafkaCluster kafkaCluster = KafkaCluster.fromCrd(kafka, VERSIONS, null);
-
-        // Mock the Kafka Operator
-        CrdOperator mockKafkaOps = supplier.kafkaOperator;
-        when(mockKafkaOps.getAsync(eq(namespace), eq(clusterName))).thenReturn(Future.succeededFuture(getKafkaCrd()));
-        when(mockKafkaOps.updateStatusAsync(any())).thenReturn(Future.succeededFuture());
-
-        // Mock the KafkaSetOperator
-        KafkaSetOperator mockKafkaSetOps = supplier.kafkaSetOperations;
-        when(mockKafkaSetOps.getAsync(eq(namespace), eq(KafkaCluster.kafkaClusterName(clusterName)))).thenReturn(Future.succeededFuture(kafkaCluster.generateStatefulSet(false, null, null)));
-
-        ArgumentCaptor<StatefulSet> reconcileStsCaptor = ArgumentCaptor.forClass(StatefulSet.class);
-        when(mockKafkaSetOps.reconcile(eq(namespace), eq(KafkaCluster.kafkaClusterName(clusterName)), reconcileStsCaptor.capture())).then(invocation -> {
-            return Future.succeededFuture(ReconcileResult.patched(invocation.getArgument(2)));
-        });
-
-        ArgumentCaptor<StatefulSet> maybeRollingUpdateStsCaptor = ArgumentCaptor.forClass(StatefulSet.class);
-        ArgumentCaptor<Function<Pod, String>> isPodToRestartFunctionCaptor = ArgumentCaptor.forClass(Function.class);
-        when(mockKafkaSetOps.maybeRollingUpdate(maybeRollingUpdateStsCaptor.capture(), isPodToRestartFunctionCaptor.capture())).thenReturn(Future.succeededFuture());
-
-        // Mock the ConfigMapOperator
-        ConfigMapOperator mockCmOps = supplier.configMapOperations;
-        when(mockCmOps.get(eq(namespace), eq(clusterName))).thenReturn(kafkaCluster.generateMetricsAndLogConfigMap(null));
-
-        // Mock the SecretOperator
-        SecretOperator mockSecretOps = supplier.secretOperations;
-        when(mockSecretOps.getAsync(eq(namespace), eq("my-tls-secret"))).thenReturn(Future.succeededFuture(getTlsSecret()));
-        when(mockSecretOps.getAsync(eq(namespace), eq("my-external-secret"))).thenReturn(Future.succeededFuture(getExternalSecret()));
-        when(mockSecretOps.reconcile(any(), any(), any())).then(invocation -> {
-            return Future.succeededFuture(ReconcileResult.created(invocation.getArgument(2)));
-        });
-
-        MockKafkaAssemblyOperator kao = new MockKafkaAssemblyOperator(vertx, new PlatformFeaturesAvailability(false, kubernetesVersion),
-                certManager,
-                passwordGenerator,
-                supplier,
-                config);
+//<<<<<<<HEAD
```

Comment:
I guess this is some rebasing issue? Should it be deleted or uncommented?

Reply:
Woops! Deleted